packages/shared/lib/core/profile/actions/profiles/checkAndMigrateProfiles.ts

```diff
@@ -194,7 +195,7 @@ function getNetworkIdFromOldNetworkType(networkType: 'mainnet' | 'devnet' | 'pri
         case 'mainnet':
             return NetworkId.Shimmer
         case 'devnet':
-            return NetworkId.Testnet
+            return NetworkId.ShimmerTestnet
```

Comment:
Do we have to update previous migrations too?

Reply:
I dont think its necessary, previous migrations came from 'devnet', should be fine